services/core-web/src/components/mine/Projects/Project.js

```diff
@@ -173,7 +177,13 @@ export class Project extends Component {
                   </div>
                 </LoadingWrapper>
               </Tabs.TabPane>
-
+              <Tabs.TabPane tab="Decision Package" key="decision-package">
+                <LoadingWrapper condition={this.state.isLoaded}>
+                  <div className="padding-lg">
+                    <DecisionPackageTab initialValues={{}} />
```

Comment:
is an empty initial values needed here?

Reply:
I was just doing some additional work setting up this feature for when the API work is done, it isn't required right now to function but it will prompt the next dev on how to easily hydrate the form I setup in the view.

Reply:
Ahhh OK thanks.